catapult/release.py

```diff
@@ -382,20 +423,26 @@ def new(
 @utils.require_2fa
 def find(_, name, commit=None):
     """
-    Search a release from the commit hash.
+    Find the first release containing a specific commit.
     """
+    if commit is None:
+        commit = "HEAD"
+
     repo = utils.git_repo()
-    commit = commit or next(utils.git_log(repo)).hex
+    oid = utils.revparse(repo, commit)
 
     client = utils.s3_client()
 
     releases = {release.commit: release for release in get_releases(client, name)}
+    print(len(releases))
 
     release = None
     for log in utils.git_log(repo):
-        release = releases.get(log.hex, release)
+        print(oid.hex, release and release.commit, log.hex)
```

Comment:
is it  a debugging leftover?

Reply:
:man_facepalming:

Reply:
Luckily I didn't leave any `import pdb`s anywhere!